src/styles/navigation.scss

```
@@ -0,0 +1,55 @@
+.navigation {
+  margin-right: 50px;
+  font-family: "Roboto", sans-serif;
+
+  &__items-list {
+    display: flex;
+    align-items: center;
+    height: 60px;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+
+    &__item {
+      margin: 0 10px;
+    }
+  }
+
+  &__link {
+    padding: 23px 0;
+
+    font-weight: 500;
+    line-height: 14px;
+    color: #000;
+    text-decoration: none;
+    text-transform: uppercase;
+  }
+
+  &__item:first-child {
+    margin-left: 0;
+  }
+
+  &__item:last-child {
+    margin-right: 0;
+  }
+
+  .is-active {
+    color: #00acdc;
+  }
+
+  .is-active {
+    position: relative;
+  }
+
+  .is-active::after {
+    display: block;
+    position: absolute;
+    height: 4px;
+    width: 100%;
+    bottom: 0;
+
+    border-radius: 8px;
+    content: "";
+    background-color: #00acdc;
+  }
```

Unify `is-active` selector entries
```suggestion
  .is-active {
    color: #00acdc;
    position: relative;
    
    &::after {
      display: block;
      position: absolute;
      height: 4px;
      width: 100%;
      bottom: 0;
      border-radius: 8px;
      content: "";
      background-color: #00acdc;
    }
  }
```